components/ProjectCard.tsx

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+import Link from "next/link";
+import { FaStar } from "react-icons/fa";
+
+const ProjectCard = ({
+  full_name,
+  url,
+  owner,
+  description,
+}: {
+  full_name: string;
+  url: string;
+  owner: any;
+  description: string;
+}) => {
+  return (
+    <div className="bg-white-900/60 dark:border-white-700 flex h-full w-full flex-col justify-between rounded-lg border p-5 shadow duration-200 delay-100">
+      <div>
+        <div className="flex flex-row">
+          {/*<img src={owner?.avatar_url} className="rounded h-40 w-40"/>*/}
+          <p>
+            <span className="text-black-500 font-semibold">{owner.login}/</span>
```

Comment:
```suggestion

```
You should remove this because `full_name` already has `WebXDAO/<repos_name>`, so it will look  `WebXDAO/<repos_name>` instead of  `WebXDAO/WebXDAO/<repos_name>`